### PR TITLE
fix: add version for `center`

### PR DESCRIPTION
**Pull request by susiwen8**

### CODE REVIEW

File: en/option/series/graph.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ Graph is a diagram to represent [nodes](~series-graph.nodes) and the [links](~se
 <ExampleUIControlVector default="0,0" dims="x,y" />
 
 Center of current view-port. It can be an arrary containing two `number`s in pixels or `string`s in percentage relative to the container width/height.
+`string` support from version `5.3.3`.
```
Comment (plainheart) [inline en/option/series/graph.md:39]: It should be in passive voice.

Comment (susiwen8) [reply en/option/series/graph.md]: Thanks, I have changed it

File: en/option/series/graph.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ Graph is a diagram to represent [nodes](~series-graph.nodes) and the [links](~se
 <ExampleUIControlVector default="0,0" dims="x,y" />
 
 Center of current view-port. It can be an arrary containing two `number`s in pixels or `string`s in percentage relative to the container width/height.
+`string` was supported from version `5.3.3`.
```
Comment (plainheart) [inline en/option/series/graph.md:39]: I think it's good to use `is` rather than `was`.

Comment (susiwen8) [reply en/option/series/graph.md:39]: Right

**Review by plainheart — APPROVED** [review]

File: zh/option/series/tree.md [hunk 3]
```diff
@@ -81,6 +81,7 @@ const option = {
 ## center(Array.<number|string>)
 
 当前视角的中心点。可以是包含两个 `number` 类型（表示像素值）或 `string` 类型（表示相对容器的百分比）的数组。
+从 `5.3.3` 版本开始支持 `string` 类型.
```
Comment (plainheart) [inline zh/option/series/tree.md:84]: `.` -> `。`

**Review by plainheart — APPROVED** [review]

**Pull request merged**